### PR TITLE
#302 Use appveyor as ci tool

### PR DESCRIPTION
**Pull request by mohamednizar**
Appveyor config was added and tested, need to create and add the project to the Appveyor account to get notifications.

### CODE REVIEW

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#345](https://github.com/yegor256/jpeek/issues/345) is now in scope, role is `REV`

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) everybody who has role `REV` is banned at [#345](https://github.com/yegor256/jpeek/issues/345); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by fabriciofx** [comment]
@mohamednizar I can't review this PR because @0crat bot doesn't select me as the `REV`.

**Comment by 0crat** [comment]
Are you speaking to me or about me [here](https://github.com/yegor256/jpeek/issues/345#issuecomment-538430660); you must always start your message with my name if you want to address it to me, see [§1](http://www.zerocracy.com/policy.html#1)

**Comment by mohamednizar** [comment]
@fabriciofx , Seems nobody assigned as `REV `, @paulodamaso/z need to assign some one to review this.

**Comment by fabriciofx** [comment]
@mohamednizar I'm waiting @paulodamaso decision.

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - cmd: java -version
+build_script:
+  - mvn clean site -Psite --errors --batch-mode
+  - mvn clean install -Pqulice --errors --batch-mode
+
+cache:
+  - C:\maven\ -> appveyor.yml
+  - C:\Users\appveyor\.m2\ -> pom.xml
+
+notifications:
```
Comment (paulodamaso) [inline .appveyor.yml]: @mohamednizar I think that we can suppress this, I think it's better to set them on Github Web INterface

Comment (mohamednizar) [reply .appveyor.yml]: @paulodamaso Perfect, GitHub Support for Web interface notification,See the image given below.  I have remove the notifications parameters. 

image

**Comment by 0crat** [comment]
This pull request [#345](https://github.com/yegor256/jpeek/issues/345) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/617df4ab-c8be-4ab7-9286-3268326681bb); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by mohamednizar** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/jpeek/issues/345#issuecomment-538962946))

@mohamednizar This is what I know about this job in [C7JGJ00DP](https://www.0crat.com/p/C7JGJ00DP), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#345](https://github.com/yegor256/jpeek/issues/345) is [in scope](http://www.zerocracy.com/policy.html#14) for 3days
  * The role is `REV`
  * The job is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) for [2days](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @mohamednizar[/z](https://www.0crat.com/u/mohamednizar): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C7JGJ00DP?q=%7Bjob%3A%27gh%3Ayegor256%2Fjpeek%23345%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by fabriciofx** [comment]
@mohamednizar @paulodamaso it seems fine to me. Can you merge it?

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job [#345](https://github.com/yegor256/jpeek/issues/345) is already in the agenda of @fabriciofx[/z](https://www.0crat.com/u/fabriciofx)

**Comment by 0crat** [comment]
The job [#345](https://github.com/yegor256/jpeek/issues/345) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by mohamednizar** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/jpeek/issues/345#issuecomment-544624090))

@mohamednizar This is what I know about this job in [C7JGJ00DP](https://www.0crat.com/p/C7JGJ00DP), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#345](https://github.com/yegor256/jpeek/issues/345) is not in scope
  * The job is waiting quality review verdict by @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) for 13days
  * @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) will receive 15 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/C7JGJ00DP?q=%7Bjob%3A%27gh%3Ayegor256%2Fjpeek%23345%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by paulodamaso** [comment]
@fabriciofx ping

**Comment by mohamednizar** [comment]
@0crat wait

**Comment by 0crat** [comment]
> @0crat wait ([here](https://github.com/yegor256/jpeek/issues/345#issuecomment-546623926))

@mohamednizar Job `gh:yegor256/jpeek#345` is not assigned, can't get performer

**Comment by fabriciofx** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/jpeek/issues/345#issuecomment-552007031))

@fabriciofx This is what I know about this job in [C7JGJ00DP](https://www.0crat.com/p/C7JGJ00DP), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#345](https://github.com/yegor256/jpeek/issues/345) is not in scope
  * The job is waiting quality review verdict by @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) for 1mon
  * @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) will receive 15 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/C7JGJ00DP?q=%7Bjob%3A%27gh%3Ayegor256%2Fjpeek%23345%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by fabriciofx** [comment]
@0crat quality is good

**Comment by 0crat** [comment]
> @0crat quality is good ([here](https://github.com/yegor256/jpeek/issues/345#issuecomment-552008206))

@fabriciofx Argument "is" doesn't match regex `good|bad|acceptable`

**Comment by fabriciofx** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx)

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx)

**Pull request merged**